build.assets/Dockerfile

```diff
@@ -250,6 +250,18 @@ RUN go install github.com/daixiang0/gci@v0.8.2
 # Install golangci-lint.
 RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1
 
+# Install Nodejs
```

Comment:
@codingllama recently updated the dockerfiles to ensure that the larger and less frequently changing layers get built first.

Let's check with him whether he plans on backporting those changes. It's probably not a big deal since the release branches are used for releases and not daily dev work.

Reply:
Backported all the way to v10, landed today.

Would you mind moving this between "Rust" and "Go", around line 195, so it matches the master Dockerfile? Also, if you are backporting for v10, would you mind doing the same there as well?

Reply:
Done!